android/build.gradle

```diff
@@ -39,5 +39,11 @@ repositories {
 dependencies {
     compile 'com.facebook.react:react-native:+'
     compile 'com.carnival.sdk:carnival:5.+'
```

Comment:
Should we pin this to a specific version as we do with the iOS SDK? It's possible that we introduce something that breaks RN without us realising...

Reply:
Yeah, sounds like a plan. I'll set a specific version number when I update to the new Android release.